src/components/form/form.tsx

```diff
@@ -132,6 +132,8 @@ export default function Form({
 
     const body = form as unknown as Registration; // converts required formValues arg type to required Registration type for POST
 
+    body.courseSyllabus = body.courseSyllabus || ''; // unrequired textarea value can be undefined. If so we convert it to a blank string.
```

Comment:
can we move this check somewhere on the server side like in `registration.ts` or `airtable.ts` etc since we already do some validations there

Reply:
Yeah sure that's fine with me, wasn't sure which side to force this on but I'll do it in `registration.ts`